velox/exec/tests/UnnestTest.cpp

```diff
@@ -39,6 +39,39 @@ TEST_F(UnnestTest, basicArray) {
 
   auto op = PlanBuilder().values({vector}).unnest({"c0"}, {"c1"}).planNode();
   assertQuery(op, "SELECT c0, UNNEST(c1) FROM tmp WHERE c0 % 7 > 0");
+
+  // Test with array containing dictionary encoded elements.
```

Comment:
This test is pretty hard to read and maintain. If we want to test encodings, it is better add a test that uses Fuzzer to generate inputs.